File: templates/png.R
# Script to ...

# Load libraries
suppressPackageStartupMessages({
  require(tidyverse)
  require(ggplot2)
})

# Load environment variables from .env file
dotenv::load_dot_env(file = ".env")

# Source modules
# source("file")

# Define args interactively or accept commandArgs
if(interactive()){
  .args <-  c("input",
              "output")
} else {
  .args <- commandArgs(trailingOnly = T)
}

# -- Create plot here --


# Save png image
ggsave(tail(.args, 1),
       p,
       width = 8.5, height = 6,
       units = "in")

# Save idenfitcal pdf image
ggsave(gsub(".png", ".pdf", tail(.args, 1)),
       p,
       width = 8.5, height = 6,
       units = "in",
       useDingbats = F)

File: templates/png.py
"""
TITLE.py

Script to ...

"""

# Load libraries
import __main__ as main
import sys
import matplotlib.pyplot as plt
import seaborn as sns
from dotenv import load_dotenv

# Load environment variables from .env file
load_dotenv(dotenv_path=".env")

# Define args interactively or accept commandArgs
if not hasattr(main, "__file__"):
    argv = ["code",
            "input",
            "output"]
else:
    argv = sys.argv


p.savefig(argv[-1])
p.savefig(argv[-1].replace(".png", ".pdf"))
